system/preside-objects/admin/security/security_user_two_factor_login_record.cfc

```diff
@@ -13,5 +13,6 @@ component extends="preside.system.base.SystemPresideObject" displayName="User 2F
 
 	property name="ip_address"     type="string"   dbtype="varchar" maxLength="50"  required=true uniqueindexes="user_machine|2";
```

Comment:
I believe we can and should get rid of the IP + user agent fields here. At the very least, the unique index needs to go.